app/views/posts/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2><%= @post.title %></h2>
+<h3>By: <%= @post.user.email %></h3>
+<p>
+  <%= @post.body %>
+</p>
+<p>
+<% if current_user.email == @post.user.email %>
+  <%= link_to "Edit Post",edit_post_path(@post) %>
+<% end %>
+<%= link_to "Add Comment",new_post_comment_path(@post) %>
+</p>
+<% if @post.comments %>
+<ul>
+  <% @post.comments.each do |comment| %>
+  <li>
+    <p>
+      <%= comment.body %>
+    </p>
+  <% if comment.user %>
+    <p>
+      <%= comment.user.email %>
+    </p>
+    <p>
+    <% if current_user.email == comment.user.email %>
```

Comment:
:+1: this prevents deleting other's comments on the front end, but try doing this in your controller as well. Otherwise savvy web developers can delete/create comments on others' belalf